DEVELOPMENT.md

```diff
@@ -1,20 +1,31 @@
 ### Local Development Install
 
-#### When using public registries
+#### Create a cluster
 
 Access to a Kubernetes cluster is needed in order to install the kpack controllers.
 
+An easy way to get a cluster up and running is to use [`kind`](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+You'll also need:
+* [`go`](https://go.dev/doc/install)
+* [`pack`](https://buildpacks.io/docs/tools/pack/)
+* [`docker`](https://docs.docker.com/get-docker/)
+* [`ytt`](https://carvel.dev/ytt/docs/v0.44.0/install/)
+
+#### When using public registries
+
 ```bash
 kubectl cluster-info # ensure you have access to a cluster
-./hack/apply.sh <IMAGE/NAME> # <IMAGE/NAME> is a writable and publicly accessible location 
+docker login <registry namespace> # must be writable and publicly accessible; e.g., your Docker Hub username or gcr.io/<some-project>
```

Comment:
An image "name" such as some-docker-username/random-image causes the tests to fail